Aspire Budgeting/Facilities/Colors.swift

```diff
@@ -100,4 +100,16 @@ extension Color {
   static let purpleFondStartColor = Color(#colorLiteral(red: 1, green: 0.7346036434173584, blue: 0.8102468252182007, alpha: 1))
   static let purpleFondEndColor = Color(#colorLiteral(red: 0.6505281925201416, green: 0.3958817720413208, blue: 0.8214730620384216, alpha: 1))
   static let purpleFondShadowColor = Color(#colorLiteral(red: 0.9686274528503418, green: 0.7058823704719543, blue: 0.8117647171020508, alpha: 1))
+
+  static func fondGradientFrom(startColor: Color,
+                                 endColor: Color) -> LinearGradient {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/dd96be95231786735b815d5fd8f3089f90ff1b53/Aspire Budgeting/Facilities/Colors.swift#L105"></span>Function parameters should be aligned vertically if they’re in multiple lines in a declaration.<br />
<code>vertical_parameter_alignment</code> <code>Colors.swift:105</code></td>
    </tr>
  </tbody>
</table>